### PR TITLE
Support for opacity

### PR DESCRIPTION
**Pull request by typemytype**
fixing #533 

**todo:**

- [x] add tests

### CODE REVIEW

**Comment by typemytype** [comment]
doc test added!

**Comment by typesupply** [comment]
@justvanrossum Do you have any thoughts on this PR? I could really, really, really use this function. 😀

File: docs/content/color.rst [hunk 1]
```diff
@@ -9,6 +9,11 @@ Colors
 	color/cmykFill
 	color/cmykStroke
 
+Opacity
+-------
+
+.. autofucntion:: drawBot.opacity
```
Comment (justvanrossum) [inline docs/content/color.rst:15]:
```suggestion
.. autofunction:: drawBot.opacity
```

**Review by justvanrossum — APPROVED** [review]
Found a typo. Otherwise looks good.

**Pull request merged**